reports/wip-limits.ts

```diff
@@ -14,15 +14,39 @@ export function getDefaultConfiguration(): any {
     };
 }
 
+interface WIPLimitsLineItem {
+    name: string,
+    count: number,
+    limit: number
+}
+
+interface WIPLimitsReport {
+    name: string,
+    lineItems: WIPLimitsLineItem[]
+}
+
 // write a table using.  see the example on stringifying the check emoji - we can do the colored circle emoji
 // https://github.com/citycide/tablemark
 
 // processing the data does a js map on each items and adds data that the report rendering (generate) needs
 // we will dump the json data used to generate the reports next to the rendered report 
 // e.g. this function should look at the transition times and added wip status of yellow, red etc. 
-export function process(data: ProjectData): ProjectData {
-    // TODO: process and add age in hours
-    return data;
+export function process(projData: ProjectData): any {
+    let report = {
+        name: `# WIP limits for ${projData.name}`,
```

Comment:
this has rendering logic in it.  I think for the purpose of this report, process should return a map of stage name to status. e.g.

```javascript
{
    "Active": "Warning",
    "In Progress": "Critical",
    "New": "OK"
    ...
}
```

so this specific report would return a type (even though the interface says it can return any object).

```javascript
declare const WipData: { [key: string]: string };
```